### DISCUSSION TITLE
Multiple Prefixes/Containers

### DISCUSSION THREAD

**Discussion by trymeouteh** [waydroid/waydroid#414] [Q&A]
Can you have multiple Android prefixes/containers, similar to how with Wine you can have multiple prefixes/containers when you use PlayOnLinux or Bottles with Wine and have a bottle using Wine 5.0, another bottle using Wine 7.1, etc?

**Comment by ChrysoliteAzalea** [comment]
Sure (as long as you are going to use only one instance at a time), just put the ```/data``` directory of the one you're going to use into ```~/.local/share/waydroid/data```.

**Reply by Aworldc** [reply]
is there a way to do it at the same time? i want to run 12+ individual copies of the same app at the same time.

**Reply by mks-h** [reply]
This doesn't work. I've tried creating copies of the `/data` directory and placing or symlinking them in `~/.local/share/waydroid/data`, but they are booting infinitely (at the LOS logo). Placing an empty `/data` directory also doesn't work — Waydroid doesn't initialize it, it stays empty.
